index.js

```diff
@@ -196,6 +223,35 @@ cm.Item({
   }
 });
 
+cm.Item({
+  label: contextMenuLabel,
+  context: [
+    cm.URLContext(['*.google.com']),
+    cm.SelectorContext(getSelectors('*', true)),
+  ],
+  contentScript: contextMenuContentScript,
+  onMessage: function(url) {
+    const regex = /url=(https[^;]*)/.exec(url)[1];
```

Comment:
We need to add a `?` after the `s` in https so that we can accept http youtube urls as well. google search sometimes embeds youtube videos in the results and those links lead with `http`
``` js
const regex = /url=(https?[^;]*)/.exec(url)[1];
```